packages/ui5-application-inquirer/src/prompts/validators.ts

```diff
@@ -32,7 +32,14 @@ export function validateAppName(appName: string, targetDir: string): boolean | s
 export async function validateFioriAppProjectFolder(targetDir: string): Promise<string | boolean> {
     const appRoot = await findRootsForPath(targetDir);
     if (appRoot) {
-        return t('validators.folderContainsFioriApp', { path: appRoot.appRoot });
+        return t('validators.folderContainsFioriApp', { path: appRoot?.appRoot });
+    }
+    // Check targetDir is in CAP project sub directories
+    const isCapPath = await findCapProjectRoot(targetDir);
```

Comment:
Should we instead create a new function in project-access to determine if the path is in a CAP project. e.g. `isCapPath(targetDir)`

Reply:
Thank you, I have updated the code to use `isCapProject()`